micropython/usb/usb-device-midi/usb/device/midi.py

```diff
@@ -108,6 +108,65 @@ def note_off(self, channel, pitch, vel=0x40):
     def control_change(self, channel, controller, value):
         self.send_event(_CIN_CONTROL_CHANGE, _MIDI_CONTROL_CHANGE | channel, controller, value)
 
+    def send_sysex(self, p):
```

Comment:
Please add a short comment about what this method does and what the argument `p` is.